src/js/components/LineList/view.ts

```diff
@@ -0,0 +1,34 @@
+import { ILine } from "@/types";
+
+const lineView = (id: string, name: string, color: string) => `
+<li class="d-flex items-center py-2 relative" data-id=${id}>
+<span class="subway-line-color-dot bg-${color}"></span>
+<span class="w-100 pl-6 subway-line-list-item-name ${color}"
+  >${name}</span
+>
+<button
+  type="button"
+  class="bg-gray-50 text-gray-500 text-sm mr-1"
+  id="update"
+>
+  수정
+</button>
+<button
+  type="button"
+  class="bg-gray-50 text-gray-500 text-sm"
+  id="remove"
+>
+  삭제
+</button>
+</li>
+`;
+
+const view = (lines: ILine[]): string => {
+  return lines.length > 0
+    ? `${lines
+        .map((line) => lineView(line.id, line.name, line.color))
```

Comment:
```suggestion
        .map(lineView)
```
lineView가 line 객체를 받아서 처리할 수 있는 형태로 만들어주면 어떨까요?